modules/virt-linux-bridge-nad-port-isolation.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-linux-bridge-nad-port-isolation.adoc_{context}"]
+= Enabling port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) that run on the same virtual LAN (VLAN) can operate in isolation from one another. When you assign an IP address to the virtual bridge, the virtual bridge can act as a gateway. The Linux bridge NAD creates a virtual bridge, or _virtual switch_, between network interfaces and the physical network. The Linux bridge NAD configures port numbers on the virtual bridge so that only specific traffic can reach each port. 
+
+
+Isolating ports in this way can provide enhanced security, efficient resource use, and performance improvements for VM workloads that run on the same node. 
+
+Before you enable port isolation, you must disable hairpin traffic from reaching the isolated ports.
+
+.Prerequisites
+
+* You created Linux bridge NAD by using either the web console or the command-line interface. 
+
+.Procedure
+
+. Edit the Linux bridge NAD by setting `portIsolation` to `true`:
++
+[source,yaml]
+----
```

Comment:
I think this example should include additional configuration that are commonly used for VMs, such as `vlan`, `disableContainerInterface`, `preserveDefaultVlan`

```
      "cniVersion":"0.3.1",
      "name": "bridge-network",
      "type": "bridge",
      "bridge": "br1",
      "preserveDefaultVlan": false,
      "vlan": 100,
      "disableContainerInterface": "true", 
      "portIsolation": "true"
```

Reply:
Added and a description for `disableContainerInterface` as no description exists in the pre-existing "Creating a Linux bridge NAD by using the command line" section.